docs/devguide/create-metricset.asciidoc

```diff
@@ -275,4 +275,3 @@ documentation.
 ==== What to Do Next
 This topic provides basic steps for creating a metricset. For more details about metricsets
 and how to extend your metricset further, see <<metricset-details>>.
-
```

Comment:
Add this blank line back in...it's sometimes required depending on how files are combined, so we always include a trailing blank line.